composer.json

```diff
@@ -13,8 +13,8 @@
         "php": ">=8.1",
         "ext-json": "*",
         "composer/package-versions-deprecated": "^1.8",
-        "phpdocumentor/reflection-docblock": "^4.3 || ^5.0",
-        "phpdocumentor/type-resolver": "^1.4",
+        "phpdocumentor/reflection-docblock": "^5.4",
```

Comment:
4.3 was not allowed either way; even with `--prefer-lowest` it would use 5.0 or 5.1. So not a breaking change